tests/tp/test_tp_strategies.py

```diff
@@ -165,13 +165,22 @@ def test_tp_train(tp_degree: int, tp_strategy: str, tmp_path: Path):
     # Compare loss and expected loss for TP
     import numpy as np
     expected_tp_loss = np.array([
```

Comment:
I think this is fine btw. TP doesn't fully work until we incorporate fsdp2 anyway, and this isn't a very good test comparing against a fixed set of loss values. It would be a better test if it compared against a non TP model, with some allowed tolerance.

Reply:
Sound good, updated the value for now and we will revisit this when we add fsdp2 support soon enough